### PR TITLE
testing-suhrudh

### PR DESCRIPTION
**Pull request by SuhrudhSarathy**

### CODE REVIEW

**Comment by hardesh** [comment]
@ojitmehta123 @threewisemonkeys-as should i merge his branch in master?

File: navigation/scripts/movement.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                self.path_use = [Point(p[0], p[1]) for p in utils.transform(
+                        LineString([(p.x, p.y) for p in self.path]), self.position_transformed, self.orientation0).coords]
+            except ValueError:
+                self.path_use = [Point(p[0], p[1]) for p in utils.transform(
+                        Point([(p.x, p.y) for p in self.path]), self.position_transformed, self.orientation0).coords]
+
+        #print([(p.x, p.y) for p in self.path_use][0])        
+        self._set_goal()
+        #self.vel_update()
+        self.rate.sleep()
```
Comment (ojitmehta123) [inline navigation/scripts/movement.py:73]:
```suggestion
```

Comment (hardesh) [reply navigation/scripts/movement.py:73]: okay.. btw what's the issue with "rate.sleep()"? It's only trying to maintain the loop at a certain frequency

Comment (ojitmehta123) [reply navigation/scripts/movement.py]: Its put in a subscriber callback which is being called with the frq of the topic. If the rate of the topic is faster than the rate set in the callback there can be issues

**Comment by hardesh** [comment]
@SuhrudhSarathy I'm closing this PR.

**Pull request closed without merging**